### PR TITLE
feat: tests for remaining EVM opcodes

### PR DESCRIPTION
**Pull request by gianbelinche**
# What ❔
This PR add tests for the remaining EVM Opcodes, the code to make them work is in this [PR](https://github.com/matter-labs/era-compiler-tester/pull/25)

## Checklist

- [x] PR title corresponds to the body of PR.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: solidity/complex/interpreter/test.json [hunk 1]
```diff
@@ -10,838 +10,1404 @@
...
+            "runtime_code": ""
+        },
+        "LOG4_Template": {
+            "runtime_code" : "602560005260206000"
+        },
+        "LOG4_Full": {
+            "runtime_code" : "602560005260206000A4"
+        },
+
+        "CREATE_Before": {
```
Comment (hedgar2017) [inline solidity/complex/interpreter/test.json:1343]: Could you group these opcodes together?
Like calls, CREATEs, returns consequently.

**Review by hedgar2017 — CHANGES_REQUESTED** [review]
Thanks a lot for helping with this!

File: solidity/complex/interpreter/test.json (hunk 1 above)
Comment (gianbelinche) [reply solidity/complex/interpreter/test.json:1343]: What do you mean here? Removing the spaces between them?

Comment (hedgar2017) [reply solidity/complex/interpreter/test.json:1343]: No, grouping similar opcodes together.

**Review by hedgar2017 — APPROVED** [review]

**Pull request merged**